simulator/cluster_compute_resource_test.go

```diff
@@ -18,13 +18,17 @@ package simulator
 
 import (
 	"context"
+	"path"
 	"testing"
 
+	"github.com/stretchr/testify/require"
```

Comment:
Stick with the order as:
* builtin
* external
* inproject

so the testify/require should be before github.com/vmware/govmomi import